addon/components/new-offering.js

```diff
@@ -10,6 +11,10 @@ export default Component.extend({
   courseStartDate: null,
   courseEndDate: null,
   smallGroupMode: true,
+  didRender() {
+    this._super(...arguments);
+    scrollTo('.offering-form .buttons', 1000);
```

Comment:
`scrollTo` is kind of a hack and will have problems with Octane. Instead use this directly with 
```
import scrollIntoView from 'scroll-into-view';
scrollIntoView(this.element);
```